pkg/scheduler/plugins/proportion/proportion.go

```diff
@@ -162,18 +163,21 @@ func (pp *proportionPlugin) OnSessionOpen(ssn *framework.Session) {
 			attr.deserved.Add(remaining.Clone().Multi(float64(attr.weight) / float64(totalWeight)))
 
 			if attr.capability != nil && !attr.deserved.LessEqualStrict(attr.capability) {
-				attr.deserved = helpers.Min(attr.deserved, attr.capability)
-				attr.deserved = helpers.Min(attr.deserved, attr.request)
```

Comment:
does it conflict with https://github.com/volcano-sh/volcano/pull/1550
which one should be reviewed? /cc @Thor-wl

Reply:
> does it conflict with #1550
> which one should be reviewed? /cc @Thor-wl

PR1550 will be reconstructed after https://github.com/volcano-sh/volcano/issues/1525 is reviewed and settled. But these modifications should be given more explaination.